### PR TITLE
Expose public path in workflow env

### PR DESCRIPTION
**Pull request by ovineq**
In order for next to build on github actions

### CODE REVIEW

File: .github/workflows/checks.yaml [hunk 1]
```diff
@@ -56,3 +56,5 @@ jobs:
         run: |
           npm run seed:mock:all
           npm run build
+        env:
```
Comment (aronhoyer) [inline .github/workflows/checks.yaml:59]: doesn't this need to be set **before** `run`? 🤔

Comment (ovineq) [reply .github/workflows/checks.yaml]: Didn't think it mattered as long as it's in the same scope/indentation. But testing now. 👍🏻

**Comment by ovineq** [comment]
> I feel like we should find a different solution to this and `.env` should be added to `.gitignore`. Allowing the file to be pushed to the remote opens up for someone accidentally committing their env config.
> 
> @nilsml had some experience with reading next js env vars from a different project, so maybe link up with him smile

Removed the `.env` and added it to the ignore list. 
Using localhost as the environment variable works, and should suffice for this workflow.
We can revisit the usage of secrets when we're setting up environments.

**Review by aronhoyer — APPROVED** [review]
looks great!

**Pull request merged**